packages/blaze/view.js

```diff
@@ -227,7 +227,7 @@ Blaze.View.prototype._errorIfShouldntCallSubscribe = function () {
  */
 Blaze.View.prototype.subscribe = function (args, options) {
   var self = this;
-  options = {} || options;
+  options = options || {};
```

Comment:
@stubailo Am I correct that in `Meteor@1.1.0.3` there is still `options = {} || options;`? If that's the case, what's the first release version this fix was merged to?